public/javascripts/main.js

```diff
@@ -0,0 +1,13 @@
+var calculator = {add: function(x, y) {
+                    return (x + y);
```

Comment:
Dang girl, your indentation is bizonkers.  Please figure out how to get sublime to indent with 2 spaces.